spring-example-product/src/main/java/com/apollographql/federation/product/App.java

```diff
@@ -0,0 +1,19 @@
+package com.apollographql.federation.product;
```

Comment:
Could you please make sure that every subproject (eg `spring-example-product`) has its own package (eg `com.apollographql.federation.springexample.product`, no strong opinion on last level of nesting)?